### PR TITLE
Bugsmasher email

### PR DESCRIPTION
**Pull request by wjbush**
Email modifications

### CODE REVIEW

**Review by claesmk — CHANGES_REQUESTED** [review]
I think your code looks OK however you have the wrong branch selected - the way the pull request is set now it's going to the main branch.  You want it to go to the bug_smashers branch.  If you edit your pull request you can change it to the correct branch.

**Comment by wjbush** [comment]
I believe the issues are now fixed.

File: test_pii_data.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def test_has_us_phone(self):
         self.assertFalse(test_data.has_us_phone())
 
     def test_has_email(self):
-        test_data = Pii()
+        test_data = Pii('My email is kavondean@gmail.com')
         self.assertEqual(test_data.has_email(), None)
```
Comment (claesmk) [inline test_pii_data.py:54]: Does the unit test run OK?  This doesn't look right, I would expect your has_email() to return True not None

**Review by claesmk — COMMENTED** [review]
Please double-check and make sure your unit test runs OK

**Review by claesmk — APPROVED** [review]
Looks good thanks for fixing the unit tests

**Comment by claesmk** [comment]
Reminder to @Terry0253 to review and approve this commit after that @wjbush you will be able to merge this pull request

**Review by Terry0253 — APPROVED** [review]
Looks good

**Pull request merged**